### PR TITLE
Ionis kutrolli development comment bug fixes, username and timestamp 

### PR DESCRIPTION
**Pull request by Ionis-Kutrolli**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -69,19 +72,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
-    long timestamp = System.currentTimeMillis();
-
-    Entity commentEntity = new Entity("Comment");
-    commentEntity.setProperty("comment", comment);
-    commentEntity.setProperty("timestamp", timestamp);
+    int maxComments = getMaxComments(request);
 
-    // TODO: setup non null users
-    commentEntity.setProperty("user", null);
+    if (maxComments == -1) {
```
Comment (yutaow) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:77]: Do you mean to check maxComments between 5 and 10?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -52,12 +52,14 @@ function loadComments() {
...
+/** Adds comments with user name to DOM. */
 function addCommentsToDOM(comments) {
     const commentListElement = document.getElementById('comment-container');
     comments.forEach((comment) => {
-      commentListElement.appendChild(createElementList(comment.comment));
-      console.log(comment.timestamp);
+      var time = new Date(comment.timestamp);
+      var commentDisplay = time.toLocaleString() + ":\t" + comment.user +  ": " + comment.comment;
+      commentListElement.appendChild(createElementList(commentDisplay));
+
```
Comment (yutaow) [inline portfolio/src/main/webapp/script.js:62]: Why empty line here?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -69,19 +72,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+      response.setContentType("text/html");
+      response.getWriter().println("Please enter integer between 5 and 10.");
+      return;
+    }
 
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(commentEntity);
```
Comment (yutaow) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:84]: Where's the code to store the new comment now?

File: portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    commentEntity.setProperty("comment", comment);
+    commentEntity.setProperty("timestamp", timestamp);
+    commentEntity.setProperty("user", user);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+    response.sendRedirect("/index.html");
+  }
+}
```
Comment (davidgaskins) [inline portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java:33]: nit: new line at end please

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (davidgaskins) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:77]: +1. Given the new servlet, maybe this method can be completely removed?

**Pull request merged**